rotkehlchen/db/filtering.py

```diff
@@ -852,13 +852,20 @@ def make(
 
 @dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
 class DBNullFilter(DBFilter):
-    columns: list[str]
+    null_columns: Optional[list[str]] = None
```

Comment:
Why this change? Isn't it more expressive to just have a different filter for the not null? Do you at any point combine those? So have something with both null_columns and not_null_columns?

Reply:
No, I don't combine them in the current filters. Will then split then it in two

Reply:
makes sense. Thanks!